packages/document/main-doc/docs/en/guides/basic-features/html.mdx

```diff
@@ -189,44 +189,44 @@ In the application root directory, create the `config/html/` directory, which su
 </html>
 ```
 
-HTML Fragments support the use [Lodash template](https://lodash.com/docs/4.17.15#template).
+HTML fragments support the use of [Lodash template](https://lodash.com/docs/4.17.15#template).
 
-For example, insert a script in `body.html`:
+For example, to insert an external script in `body.html`:
 
 ```html title="config/html/body.html"
 <script src="//example.com/assets/a.js"></script>
 ```
 
 :::info
-The implementation of the custom HTML fragment is to merge the fragment with the built-in template of the frame. Since `<title>` already exists in the default template of the frame, the `<title>` in the custom HTML template cannot take effect. Please pass [html.title](/configure/app/html/title) to modify the page title.
+The implementation of custom HTML fragments is to merge the fragments with the framework's built-in template. Since the `<title>` already exists in the default template of the framework, the title tag in the custom HTML template will not take effect. Please use [html.title](/configure/app/html/title) to modify the page title.
 
 :::
 
-### Fully Custom HTML Templates
+### Custom All HTML Templates
 
-In some cases, HTML snippets do not meet the customization requirements, Modern.js provide a fully customized way.
+In some cases, HTML fragments cannot meet the customization requirements. Modern.js provides a fully customized way.
 
-:::caution
-It is not recommended to override the default HTML template directly, and some features may be lost. Even if it needs to be replaced, it is recommended to build on the built-in template and modify it as needed.
+:::caution Note
+It is generally not recommended to directly override the default HTML template, as some functional options may be lost. Even if replacement is required, it is recommended to modify based on the built-in template as needed.
 
 :::
 
-In the `config/html/` directory, create a index.html file that will override the default HTML template.
+Under the `config/html/` directory, create an `index.html` file that will replace the default HTML template.
 
 :::info
-The internal default HTML template can be viewed in `node_modules/.modern-js/${entryName}/index.html`.
+The default HTML template can be viewed in `node_modules/.modern-js/${entryName}/index.html`.
 
 :::
 
-### Template Params
+### Template Parameters
 
-The parameters used in the template can be defined by the [html.templateParameters](/configure/app/html/template-parameters).
+The parameters used in the template can be defined by the [html.templateParameters](/configure/app/html/template-parameters) configuration.
```

Comment:
can be defined by -> are the same as

Reply:
This means that it can be used to define variables in html.templateParameters, not this same as